apps/btc_family/btc_api.c

```diff
@@ -108,27 +108,22 @@ bool decode_btc_query(const uint8_t *data,
     return false;
   }
 
-  /* Initialize bitcoin query */
-  btc_query_t query = BTC_QUERY_INIT_ZERO;
-
   /* Create a stream that reads from the buffer. */
   pb_istream_t stream = pb_istream_from_buffer(data, data_size);
 
   /* Now we are ready to decode the message. */
-  bool status = pb_decode(&stream, BTC_QUERY_FIELDS, &query);
+  bool status = pb_decode(&stream, BTC_QUERY_FIELDS, query_out);
```

Comment:
Let's zeroise `query_out` buffer before decoding to be safe

Reply:
added